From: KevinDuan <duankaiwen@kylinos.com>
Date: Tue, 21 May 2024 02:28:10 +0000
Subject: Translated using Weblate (Uyghur)

Currently translated at 100.0% (3 of 3 strings)

Translation: openkylin-nile-new/kylin-os-manager
Translate-URL: http://weblate.openkylin.top/projects/openkylin-nile-new/kylin-os-manager/ug/
---
 src/application/translations/kylin-os-manager_ug.ts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/application/translations/kylin-os-manager_ug.ts b/src/application/translations/kylin-os-manager_ug.ts
index eebd88e..fa272fa 100644
--- a/src/application/translations/kylin-os-manager_ug.ts
+++ b/src/application/translations/kylin-os-manager_ug.ts
@@ -13,12 +13,12 @@
     <message>
         <location filename="../mainwindow.cpp" line="224"/>
         <source>kylin os manager</source>
-        <translation type="unfinished"></translation>
+        <translation>Kylin os باشقۇرغۇ</translation>
     </message>
     <message>
         <location filename="../mainwindow.cpp" line="225"/>
         <source>version: %1</source>
-        <translation type="unfinished"></translation>
+        <translation>نەشرى: ٪1</translation>
     </message>
 </context>
 </TS>
